src/components/PersonaCard/EditAndRemoveMenu.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Dropdown, Menu } from 'antd';
+import styled from 'styled-components';
+import EditIcon from 'assets/EditIcon.svg';
+import RemoveIcon from 'assets/RemoveIcon.svg';
+import EditMenu from 'assets/EditMenu.svg';
+import { NavLink, useLocation } from 'react-router-dom';
+import { getPathnameArray } from 'helpers/url';
+
+const EditMenuBox = styled.div`
+  position: relative;
+  float: right;
+  margin-top: 8px;
+`;
+const EditAndRemoveBox = styled.div`
+  ${props => props.theme.typography.caption};
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 84px;
+`;
+
+const MenuStyled = styled(Menu)`
+  padding: 0px;
+  border-radius: 0;
+`;
+
+const MenuEditStyled = styled(Menu.Item)`
+  border-bottom: 1px ${props => props.theme.colors.main.primary} solid;
+  padding: 4px 4px 4px 8px;
+`;
+
+const MenuRemoveStyled = styled(Menu.Item)`
+  border-top: 1px ${props => props.theme.colors.main.primary} solid;
+  padding: 4px 4px 4px 8px;
+`;
+
+const NavLinkStyled = styled(NavLink)`
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.65);
+`;
+type EditAndRemoveMenuType = {
+  uuid: string;
+};
+
+export const EditRemoveMenu = ({ uuid }: EditAndRemoveMenuType) => {
```

Comment:
EditRemoveMenu should be typed as FC - Functional Component

Reply:
I cant because when i have FC i receive error. Fc is unnecessary in this case